### PR TITLE
Dev loll: stability fix

### PR DESCRIPTION
**Pull request by lollcat**
Fix stability issue with FAB without the replay buffer.
Specifically: we have to recompute log_q(x) in the loss instead of re-using it to prevent issues with Nans. 
This bug was introduced in #59 and only effects extreme .

### CODE REVIEW

File: fab/sampling_methods/base.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def create_point(x: torch.Tensor, log_q_fn: LogProbFunc, log_p_fn: LogProbFunc,
     else:
         # Use log_q_x if we already have it, otherwise calculate it.
         log_q_x = log_q_x if log_q_x is not None else log_q_fn(x)
-        return Point(x=x, log_q=log_q_x, log_p=log_p_fn(x))
+        return Point(x=x, log_q=log_q_x.detach(), log_p=log_p_fn(x).detach())
```
Comment (lollcat) [inline fab/sampling_methods/base.py:72]: No longer re-using log_q_x in the loss function (as if it contains NaNs then this causes numerical instability)

File: fab/core.py [hunk 2]
```diff
@@ -186,15 +183,6 @@ def forward_kl(self, x_p: torch.Tensor) -> torch.Tensor:
         """Forward kl with estimated using x ~ p(x) where p is the target distribution."""
         return -torch.mean(self.flow.log_prob(x_p))
 
-    def fab_forward_kl_inner(self, point: Point, log_w_ais: torch.Tensor) -> torch.Tensor:
```
Comment (lollcat) [inline fab/core.py:189]: Now can be done with alpha=1 so removing
